hi-ml/src/health_ml/runner.py

```diff
@@ -268,7 +268,6 @@ def after_submission_hook(azure_run: Run, *args: Any) -> None:
                 after_submission=after_submission_hook,
                 tags=self.additional_run_tags(script_params),
                 strictly_aml_v1=self.experiment_config.strictly_aml_v1,
-                processes_per_node_v2=self.lightning_container.num_gpus_per_node(),
```

Comment:
Should this arg have been renamed to "pytorch_processes_per_node_v2" rather than removed completely?

Reply:
No, it should be removed completely, we want an MPI job rather than a plain Pytorch job like you are running.

Reply:
So it will not be possible to run a plain pytorch multi-gpu job using the runner script?

Reply:
No, it will not be possible. The runner is not using plain PyTorch, but PL. So it's more: You shouldn't even attempt to run a plain pytorch job with the runner.